From 75e3854046edb1c64f22b951f9b16898831ac905 Mon Sep 17 00:00:00 2001
From: Guo Yi <yguo@cavium.com>
Date: Fri, 18 Dec 2020 00:46:18 +0000
Subject: [PATCH 1453/1921] spi: orion: add compatible string for cp110

Add a new compatible string for cp11x for special maximum speed setting

Signed-off-by: Guo Yi <yguo@cavium.com>
Change-Id: I11298ad0bd007a4b4a341a931423a5d0e9105c6a
Reviewed-on: https://sj1git1.cavium.com/c/IP/SW/kernel/linux/+/42480
Tested-by: sa_ip-sw-jenkins <sa_ip-sw-jenkins@marvell.com>
Reviewed-by: Kostya Porotchkin <kostap@marvell.com>
Reviewed-by: Nadav Haklai <nadavh@marvell.com>
Reviewed-on: https://sj1git1.cavium.com/c/IP/SW/kernel/linux/+/50162
Reviewed-by: Stefan Chulski <stefanc@marvell.com>
[WK: The original patch got from Marvell sdk11.21.09]
Signed-off-by: Wenlin Kang <wenlin.kang@windriver.com>
---
 Documentation/devicetree/bindings/spi/spi-orion.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Documentation/devicetree/bindings/spi/spi-orion.txt b/Documentation/devicetree/bindings/spi/spi-orion.txt
index 8434a65fc12a..2fe873402d99 100644
--- a/Documentation/devicetree/bindings/spi/spi-orion.txt
+++ b/Documentation/devicetree/bindings/spi/spi-orion.txt
@@ -7,6 +7,7 @@ Required properties:
     - "marvell,armada-375-spi", for the Armada 375 SoCs
     - "marvell,armada-380-spi", for the Armada 38x SoCs
     - "marvell,armada-390-spi", for the Armada 39x SoCs
+    - "marvell,armada-cp110-spi", for the Armada 7K/8K and Octeon TX2 CN913x SoCs
     - "marvell,armada-xp-spi", for the Armada XP SoCs
 - reg : offset and length of the register set for the device.
 	This property can optionally have additional entries to configure
-- 
2.31.1

